documentation/user_docs/docs/user_guide/Getting_started.rst

```diff
@@ -482,85 +655,201 @@ This should give a plot that looks like this:
+any dimensionality of dataset, so you typically write functions such as this
+that work only for a particular dimensionality. It is important, therefore, for
+your own book-keeping, that you give the functions sensible names that reflect
+both what they do and what sort of dataset they apply to.
+
+Now let's run the function, here we will use ``func_eval``.
 
-Now let's run the function. Instead of using ``user_func`` we will use ``func_eval``. The syntax for functions called by
-this routine is slightly different:
+The syntax for functions called by this routine is slightly different:
 
 ::
 
-   w_sim= func_eval (w_template,@demo_4gauss,[6 1 1 0.1 1.25 6 1]);
+   w_sim = func_eval(w_template, @demo_4gauss, [6 1 1 0.1 1.25 6 1]);
 
 
-The arguments in the square parentheses are the function inputs, and in this case they correspond respectively to
-amplitude, satellite position x-coordinate, satellite y-coordinate, central position x-coordinate, central y-coordinate,
-and background. In general the input to a function called by ``func_eval`` can take any form (e.g. a cell array, a
-structure array, a string, etc.), although if you wish to pass anything other than a vector of parameters, such as that
-shown above, then it must be packed into a cell array.
+The arguments in the square parentheses are the function inputs, and in this
+case they correspond respectively to:
 
-Notice that the syntax of the input arguments is somewhat different for ``func_eval`` compared to ``user_func``, since
-with the former we input the parameters as a vector, rather than as separate arguments. The form of the function itself
-is also different, since it takes some arrays of parameters and calculates an intensity at those points, rather than
-taking an existing intensity array and modifying it.
+- amplitude
+- satellite position x-coordinate
+- satellite y-coordinate
+- central position x-coordinate
+- central y-coordinate
+- background
+
+.. note::
+
+   In general the input to a function called by ``func_eval`` can take any form
+   (e.g. a cell array, a structure array, a string, etc.), although if you wish
+   to pass anything other than a vector of parameters, such as that shown above,
+   then it must be packed into a cell array.
 
 .. image:: ../images/Screenshot_SimCut.png
    :width: 300px
    :alt: 2d simulation
 
 
-``func_eval`` works for both sqw and dnd objects with almost the same syntax. For sqw objects pixel information is
-simulated according to the intensity calculated for the data grid, whereas for dnd objects this is not required. It is
-also possible to simulate a dnd from a template sqw object by using an additional keyword argument of the form
+.. note::
+
+   ``func_eval`` works for both ``sqw`` and ``dnd`` objects with almost the same
+   syntax.
+
+
+.. note::
+
+   For ``sqw`` objects, pixel information is simulated according to the
+   calculated intensity for the data grid, whereas for ``dnd`` objects this is
+   not required.
```

Comment:
I don't understand this. What does simulation according to the calculated intensity? Is this something about the grid? What would happen for a dnd?